tests/src/Common/PerfHarness/PerfHarness.csproj

```diff
@@ -6,9 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.performance.api">
-      <Version>1.0.0-beta-build0006</Version>
-    </PackageReference>
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0007" />
```

Comment:
@weshaggard Is there a way to import `dir.props` and `dir.targets` in the new `csproj` file? I would like to have the version of the package in a central place. Is that a supported scenario?